src/components/Permissions/AppPermissions/AppPermissions.spec.jsx

```diff
@@ -172,18 +172,24 @@ describe('AppPermissions', () => {
     expect(queryByTestId('Spinner')).toBeTruthy()
   })
 
-  it('should display slugName when query is not loading', () => {
+  it('should display appName when query is not loading', () => {
     useAppsOrKonnectorsBySlug.mockReturnValue({
       isResultLoading: false,
       hasQueryFailed: false,
-      result: { fetchStatus: 'loaded', data: {} }
+      result: {
+        fetchStatus: 'loaded',
+        data: {
+          name: 'Drive',
+          name_prefix: 'Coz'
+        }
+      }
     })
     const { queryByText } = render(
       <BreakpointsProvider>
         <AppPermissions />
       </BreakpointsProvider>
     )
-    expect(queryByText('DRIVE')).toBeInTheDocument()
+    expect(queryByText('Coz Drive'.toUpperCase())).toBeInTheDocument()
```

Comment:
nit: maybe we can use toBeTruthy instead toBeInTheDocument ?

Reply:
> nit: maybe we can use toBeTruthy instead toBeInTheDocument ?

There is a lot of things that are not OK within this test file. `toBeIntheDocument` is the latest thing I'll change ^^'. 

For instance, tests are not isolated. This one need the first to be run otherwise it fails. Mocks returned values are not OK, we don't return only data with an empty object... Lot of console.error...